ts/test/scenario.ts

```diff
@@ -346,6 +346,65 @@ test("pauseUntilDhtEqual - Create multiple entries, read the last, 2 conductors"
   await scenario.cleanUp();
 });
 
+test("dhtSync - Fails if some Ops were authored but not yet integrated in a conductor", async () => {
+  const scenario = new Scenario();
+
+  const appBundleSource: AppBundleSource = {
+    type: "path",
+    value: FIXTURE_HAPP_URL.pathname,
+  };
+  const [alice] = await scenario.addPlayersWithApps([
+    { 
+      appBundleSource,
+      options: {
+        networkConfig: {
+          disablePublish: true
```

Comment:
This is not right. I want to disable the integration workflow so that an Op is authored but not integrated, and I confirm that dhtSync fails. Is that possible some other way?